delta-contribs/src/main/scala/org/apache/spark/sql/delta/storage/COSLogStore.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (2020) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta.storage
+
+import com.google.common.base.Throwables
+
+import java.io.IOException
+import java.nio.charset.StandardCharsets.UTF_8
+import java.nio.file.FileAlreadyExistsException
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.SparkConf
+
+/**
+ * LogStore implementation for IBM Cloud Object Storage.
+ *
+ * We assume the following from COS's [[FileSystem]] implementations:
+ * - Write on COS is all-or-nothing, whether overwrite or not.
+ * - Write is atomic.
+ *   Note: Write is atomic when using the Stocator v1.1.1+ - Storage Connector for Apache Spark
+ *   (https://github.com/CODAIT/stocator) by setting the configuration `fs.cos.atomic.write` to true
+ *   (for more info see the documentation for Stocator)
+ * - List-after-write is consistent.
+ */
+class COSLogStore(sparkConf: SparkConf, hadoopConf: Configuration)
```

Comment:
Is COSLogStore the best name for this? Typically we are moving towards the putting the cloud name in the logStore (like AzureLogStore) because 
(i) usually the cloud name is more easily recognizable than the specific storage system name. 
(ii) usually a cloud has a single scalable storage solution and there fore one log store implementation is sufficient for the cloud (Azure has multiple storage solutions but they have same semantics so luckily one logstore implementation was good enough)

So in this case, wouldn't it be more recognizable in general if we put IBM in the name. like "IBMLogStore" or "IBMCOSLogStore"?